### PR TITLE
Making changes for publishing nimble collection in Ansible collection…

### PR DESCRIPTION
**Pull request by ar-india**
Making changes for publishing nimble collection in Ansible collection

The details guidlines are give in the below link https://github.com/ansible-collections/overview/blob/main/collection_requirements.rst#overview.

One of the basic requirement is that the sanity test should pass. in order to achieve that we need to push in the test/sanity/ignore* file.

Signed-off-by: ar-india <alokranjan2005@gmail.com>

### CODE REVIEW

**Comment by ar-india** [comment]
sanity test result:
[root@centos80x64tmp nimble]# pwd
/root/.ansible/collections/ansible_collections/hpe/nimble

[root@centos80x64tmp nimble]# ansible-test sanity --docker
Emulate Docker CLI using podman. Create /etc/containers/nodocker to quiet msg.
Emulate Docker CLI using podman. Create /etc/containers/nodocker to quiet msg.
Emulate Docker CLI using podman. Create /etc/containers/nodocker to quiet msg.
Running sanity test 'action-plugin-docs' with Python 3.6
Running sanity test 'ansible-doc' with Python 3.6
Running sanity test 'changelog' with Python 3.6
Running sanity test 'compile' with Python 2.6
Running sanity test 'compile' with Python 2.7
Running sanity test 'compile' with Python 3.5
Running sanity test 'compile' with Python 3.6
Running sanity test 'compile' with Python 3.7
Running sanity test 'compile' with Python 3.8
Running sanity test 'compile' with Python 3.9
Running sanity test 'empty-init' with Python 3.6
Running sanity test 'future-import-boilerplate' with Python 3.6
Running sanity test 'ignores'
Running sanity test 'import' with Python 2.6
Running sanity test 'import' with Python 2.7
Running sanity test 'import' with Python 3.5
Running sanity test 'import' with Python 3.6
Running sanity test 'import' with Python 3.7
Running sanity test 'import' with Python 3.8
Running sanity test 'import' with Python 3.9
Running sanity test 'line-endings' with Python 3.6
Running sanity test 'metaclass-boilerplate' with Python 3.6
Running sanity test 'no-assert' with Python 3.6
Running sanity test 'no-basestring' with Python 3.6
Running sanity test 'no-dict-iteritems' with Python 3.6
Running sanity test 'no-dict-iterkeys' with Python 3.6
Running sanity test 'no-dict-itervalues' with Python 3.6
Running sanity test 'no-get-exception' with Python 3.6
Running sanity test 'no-illegal-filenames' with Python 3.6
Running sanity test 'no-main-display' with Python 3.6
Running sanity test 'no-smart-quotes' with Python 3.6
Running sanity test 'no-unicode-literals' with Python 3.6
Running sanity test 'pep8' with Python 3.6
Running sanity test 'pslint'
Running sanity test 'pylint' with Python 3.6
Running sanity test 'replace-urlopen' with Python 3.6
Running sanity test 'rstcheck' with Python 3.6
Running sanity test 'runtime-metadata' with Python 3.6
Running sanity test 'shebang' with Python 3.6
Running sanity test 'shellcheck'
Running sanity test 'symlinks' with Python 3.6
Running sanity test 'use-argspec-type-path' with Python 3.6
Running sanity test 'use-compat-six' with Python 3.6
Running sanity test 'validate-modules' with Python 3.6
Running sanity test 'yamllint' with Python 3.6
Emulate Docker CLI using podman. Create /etc/containers/nodocker to quiet msg.

File: ansible_collection/hpe/nimble/changelogs/.plugin-cache.yaml [hunk 1]
```diff
@@ -0,0 +1,125 @@
+plugins:
```
Comment (datamattsson) [inline ansible_collection/hpe/nimble/changelogs/.plugin-cache.yaml:1]: This file should not be part of the repo I think. I see other repos adding .plugin-cache.yaml to .gitignore.

File: ansible_collection/hpe/nimble/changelogs/config.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  - Deprecated Features
+- - removed_features
+  - Removed Features (previously deprecated)
+- - security_fixes
+  - Security Fixes
+- - bugfixes
+  - Bugfixes
+- - known_issues
+  - Known Issues
+title: Hpe.Nimble
```
Comment (datamattsson) [inline ansible_collection/hpe/nimble/changelogs/config.yaml]: I see other collections using the full vanity name here. "HPE Nimble Storage Content Collection for Ansible".

**Review by datamattsson — CHANGES_REQUESTED** [review]
A few nits.

File: ansible_collection/hpe/nimble/changelogs/config.yaml (hunk 2 above)
Comment (ar-india) [reply ansible_collection/hpe/nimble/changelogs/config.yaml]: ok .. will make the changes accordingly

**Comment by ar-india** [comment]
have made the changes as suggested by Michael. have also added support for execution env.

**Review by ar-india — COMMENTED** [review]
please review

**Review by datamattsson — APPROVED** [review]
LGTM, but is the plugin-cache still in the commit or not? I can't tell.

**Review by suneeth51 — APPROVED** [review]
lgtm..

File: ansible_collection/hpe/nimble/requirements.txt [hunk 3]
```diff
@@ -0,0 +1,2 @@
+ansible-pylibssh
```
Comment (anshulbehl) [inline ansible_collection/hpe/nimble/requirements.txt:1]: nimble-sdk also needs to be added?

Comment (ar-india) [reply ansible_collection/hpe/nimble/requirements.txt:1]: thanks. added the same

**Pull request merged**